tests/manage/z_cluster/nodes/test_check_pods_status_after_node_failure.py

```diff
@@ -189,12 +195,22 @@ def test_check_pods_status_after_node_failure(self, nodes, node_restart_teardown
         ), f"The new pods are not 'Running' after {timeout} seconds"
 
         log.info("All the pods are in 'Running' or 'Completed' state")
-        log.info(f"Starting the node '{node_name}' again...")
-        nodes.start_nodes(nodes=[ocs_node])
```

Comment:
wouldn't this affect the test case of the product? should there be a `if not is_managed_service_cluster()` for the product test case?

Reply:
No. I added it again here https://github.com/red-hat-storage/ocs-ci/pull/5977/files#diff-6d675cfa44bd4b51676bf7b6403c79557b1f1f28cd3172123c81392b5a013166R209, in case it is not MS.

Reply:
ah okay. thanks Itzhak.